### PR TITLE
Create uwo.yml

### PR DESCRIPTION
**Pull request by ableyjoe**
Western University Canada (formerly University of Western Ontario)

Adds or updates the following events or companies:
 - Western University
 - 

### University updates
 - [X] I have put the most recent relevant date in the "Last Update" column
 - [X] I have linked to the article about the change, not the company's website (Please put N/A if there is no public post)

### Company updates
 - [ ] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's website (Please put N/A if there is no public post)

### Event updates
 - [ ] I have linked to the article about the change, not the event's website

### CODE REVIEW

File: _data/universities/uwo.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: "[Western University](https://news.westernu.ca/2020/03/presidents-statement-on-covid-19-efforts-march-12/)"
```
Comment (drewbrew) [inline _data/universities/uwo.yml]: Would you mind renaming the file to `western-university-canada.yml` or something similar? While UWO isn't as likely to encounter collisions as, say, OSU or ASU, I'd like to keep consistent if possible.

Comment (ableyjoe) [reply _data/universities/uwo.yml]: how about naming universities using their domain names in reverse order, a bit like Apple vendor plists? So, ca.uwo.yml, edu.stanford.yml, etc. It'd avoid collisions and would also provide some visual grouping by university location.

Comment (drewbrew) [reply _data/universities/uwo.yml]: Hmm, that seems cool.

What do you think, @toothbrush and @phildini?

**Comment by drewbrew** [comment]
I'm going to merge this now and create a separate issue for university file naming. Thanks!

File: _data/universities/uwo.yml (hunk 1 above)
Comment (toothbrush) [reply _data/universities/uwo.yml]: Tough call!  If it were me, i wouldn't enforce too much, to keep people feeling there's a low barrier to contributing.

But for sure, consistency has the advantage that it's easier for folks to see if they're about to make a duplicate...  Depends on whether you want more mental burden on the maintainers or the contributors!

**Pull request merged**